File: cfgs/java_cfgs_foo/MIDDLE_OF_THREE_USING_MINIMUM_COMPARISONS.java
Nodes
1;special;;;
2;conditional;((foo_1 < foo_2) && (foo_3 < foo_4)) || ((foo_5 < foo_6) && (foo_7 < foo_8));14;
3;statement;return foo_1;14;
4;exit;;;
5;conditional;((foo_1 < foo_2) && (foo_3 < foo_4)) || ((foo_5 < foo_6) && (foo_7 < foo_8));15;
6;statement;return foo_1;15;
7;statement;return foo_1;16;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;7;
5;4;
6;4;

File: cfgs/pythoncfgs/MIDDLE_OF_THREE_USING_MINIMUM_COMPARISONS.py
Nodes
1;conditional;foo_1 < foo_2 and foo_3 < foo_4 or (foo_5 < foo_6 and foo_7 < foo_8);8;
2;conditional;foo_1 < foo_2 and foo_3 < foo_4 or (foo_5 < foo_6 and foo_7 < foo_8);10;
3;statement;return foo_1;13;
4;statement;return foo_1;11;
5;statement;return foo_1;9;
6;exit;;
Edges
0;2;5;
1;3;4;
2;6;
3;6;
4;6;
5;
